bench/record/record.bash

```diff
@@ -15,6 +15,10 @@ bench_start=`env_val "${env}" 'bench.start'`
 if [ -z "${bench_start}" ]; then
 	bench_start='0'
 fi
+test_name=`env_val "${env}" "bench.${workload}.test-name"`
```

Comment:
This module `bench.record` is for all workloads, other workload may not have `test-name` but have other props.

The best way is use score table for common info, then use detail table to record as many info as posible.